analytics/filesystem/file_module.go

```diff
@@ -85,6 +85,9 @@ func (f *FileLogger) LogNotificationEventObject(ne *analytics.NotificationEvent)
 	f.Logger.Flush()
 }
 
+// Shutdown the logger - No-op Implementation
+func (f *FileLogger) Shutdown() {}
```

Comment:
Should we call flush in order to output all pending buffered data just in case there's any? 
```
88   // Shutdown the logger - No-op Implementation
89 - func (f *FileLogger) Shutdown() {}
   + func (f *FileLogger) Shutdown() {
   +     f.Logger.Flush()
   + }
90 
91   // Method to initialize the analytic module
92   func NewFileLogger(filename string) (analytics.Module, error) {
analytics/filesystem/file_module.go
```